### PR TITLE
add motorola droid turbo support

### PR DESCRIPTION
**Pull request by 18712886438**

### CODE REVIEW

File: libmbp/devices/motorola.cpp [hunk 1]
```diff
@@ -169,6 +169,19 @@ void addMotorolaDevices(std::vector<Device *> *devices)
     device->setRecoveryBlockDevs({ BOOTDEVICE_RECOVERY, F9824900_RECOVERY,
         F9824900_SOC0_RECOVERY, "/dev/block/mmcblk0p37" });
     devices->push_back(device);
+
+    // Motorola Droid Turbo 1st
+    device = new Device();
+    device->setId("quark");
+    device->setCodenames({ "quark","quark_lra","quark_umts","quark_verizon","xt1225","xt1250","xt1254" });
```
Comment (Caio99BR) [inline libmbp/devices/motorola.cpp]: Use a space after each ','

Comment (18712886438) [inline libmbp/devices/motorola.cpp]: @Caio99BR  How about now?

Comment (Caio99BR) [inline libmbp/devices/motorola.cpp]: Ok, if you can fix 80+ char, if not ok of same way

Comment (18712886438) [inline libmbp/devices/motorola.cpp]: @Caio99BR  Sorry I am not good with English, Is there any problem ?

Comment (Caio99BR) [inline libmbp/devices/motorola.cpp]: No, its Ok
@chenxiaolong this is your time

Comment (18712886438) [inline libmbp/devices/motorola.cpp]: Ok Thx. :)

**Pull request merged**